### DISCUSSION TITLE
Merge attempt failed on PR after downtime

### DISCUSSION THREAD

**Discussion by nguyendanv** [community/community#118601] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Seeing this error when trying to merge this PR after the downtime earlier (I made an attempt to merge during the downtime which is likely the cause) - how to resolve this? 

<img width="1040" alt="Screenshot 2024-04-10 at 3 10 27 PM" src="https://github.com/community/community/assets/17125600/8c2806a3-6191-4a80-b3c1-fe62fa18e521">

Saw similar question here, but no clear resolution: https://github.com/orgs/community/discussions/118526

**Comment by nguyendanv** [comment, marked as answer]
Resolved this by creating a new branch and opening a new PR

**Reply by jonespm** [reply]
I also had this error and I was able to merge the broken PR now. Not sure if waiting ~30 minutes fixed it or something else did.
